lib/multi_factor_auth.rb

```diff
@@ -83,4 +84,11 @@ def self.send_phone_mfa(phone_number, digits: 5)
   def self.is_enabled?
     Rails.configuration.feature_flag_mfa
   end
+
+  def self.domestic_country_code(phone_number)
+    VALID_DOMESTIC_COUNTRIES.each do |country_code|
+      return country_code if TelephoneNumber.valid?(phone_number, country_code, [:mobile])
+    end
+    false
```

Comment:
I don't think this explicit `false` is needed, because `nil` is falsey (and arguable makes more sense for "no such value" in this context)

Reply:
Thanks, I've removed this explicit return value

Reply:
I've had to put it back.  Several slightly related tests fail when it isn't there.  😕